plugins/woocommerce-blocks/assets/js/blocks/mini-cart-interactivity/utils/set-styles.ts

```diff
@@ -0,0 +1,51 @@
+function getClosestColor(
```

Comment:
Copy pasted this stuff, we will refactor towards implementing things like setStyles in the interactivity block.